[UNIT] Add draft Gradio bonus module for agents-course (#120)
huggingface/agents-course: units/en/unit0/introduction.mdx

| 2 | Frameworks | Understand how the fundamentals are implemented in popular libraries : smolagents, LangGraph, LLamaIndex |
| 3 | Use Cases | Let's build some real life use cases (open to PRs 🤗 from experienced Agent builders) |
| 4 | Final Assignment | Build an agent for a selected benchmark and prove your understanding of Agents on the student leaderboard  🚀 |

> | 5 | Bonus Gradio Module | Learn to build and deploy interactive AI agents with Gradio interfaces |

Reviewer: just a suggestion, not sure if this makes more sense
```suggestion
| 5 | Bonus Gradio Module | Learn to build and deploy web applications for AI agents using Gradio |
```

> *We have one bonus unit available for you: Gradio module helps you create interactive interfaces for your agents. More bonus units coming soon!*

Reviewer 2: I would not change that sentence. As it removes the emphasis on other future bonus Units

Reviewer 3: Agreed with @Jofthomas. Because we don't have only bonus units with gradio, for instance next week we have one that is not about gradio but fine-tune.

Author: Makes sense. Thanks for the reviews.
